Winch/Core/Initializer.cs

```diff
@@ -48,5 +51,24 @@ private static void InitializeDevConsole()
             term.AddComponent<Terminal>();
             UnityEngine.Object.DontDestroyOnLoad(term);
         }
+
+        private static async void CheckForUpdate()
+        {
+            var client = new GitHubClient(new ProductHeaderValue("WinchFetchLatest"));
+            var releases = await client.Repository.Release.GetAll("Hacktix", "Winch");
+            var latest = releases[0];
+
```

Comment:
There is an 'unauthenticated rate limit' of I believe 60 requests/hour but that shouldn't be a problem.